config/example.ini

```diff
@@ -250,47 +250,34 @@ plugin = python
 #
 # Logging configuration
 #
-kinto.logging_renderer = kinto.core.logs.MozillaHekaRenderer
 
 [loggers]
-# keys = root, sentry
-keys = root, kinto
+keys = root
 
 [handlers]
 keys = console, sentry
 
 [formatters]
-keys = heka, generic
+keys = generic, json
 
 [logger_root]
 level = INFO
 handlers = console, sentry
-formatter = heka
-
-[logger_kinto]
-level = INFO
-handlers =
-qualname = kinto
-
-[logger_sentry]
-level = WARN
-handlers = console
-qualname = sentry.errors
-propagate = 0
 
 [handler_console]
 class = StreamHandler
 args = (sys.stdout,)
 level = NOTSET
```

Comment:
@Natim isn't it because of this?

Reply:
I don't see any difference even if I put it to DEBUG